omniauth-signnow.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.version       = OmniAuth::Signnow::VERSION
 
   gem.add_dependency 'omniauth', '~> 1.0'
-  gem.add_dependency 'omniauth-oauth2', '~> 1.1.1'
+  gem.add_dependency 'omniauth-oauth2', '~> 1.1'
```

Comment:
This is the important change here. It prevents any `1.x` from being used with this gem, which makes it hard to keep dependencies up to date when using it.